cpp/src/binaryop/compiled/operation.cuh

```diff
@@ -162,12 +162,24 @@ struct PMod {
     if (rem < 0) rem = std::fmod(rem + yconv, yconv);
     return rem;
   }
+
+  template <typename TypeLhs,
+            typename TypeRhs,
+            std::enable_if_t<cudf::is_fixed_point<TypeLhs>() and
+                             std::is_same_v<TypeLhs, TypeRhs>>* = nullptr>
+  __device__ inline auto operator()(TypeLhs x, TypeRhs y)
```

Comment:
I noticed that every struct in this file has trailing return types except `PMod`. Might be worth addressing in a separate PR.

Reply:
Probably better to save for another PR.